### ISSUE TITLE
MangoHUD does not display the correct GPU clock of a 7900 XTX

### ISSUE THREAD

**Issue by DoomPenguin9** [flightlessmango/MangoHud#954]
When I use MangoHUD with my RX 7900 XTX, it just shows that the GPU clock is 1200 MHz, which is wrong. Memory clock is displayed correctly.

OS: Arch Linux
Kernel: 6.2.5-zen1-1-zen
Mesa version: Mesa 23.1.0-devel (git-b190d08a8a)
Linux Firmware version: 20230210.bf4115c-1
MangoHUD version: 0.6.8.r108.g06b46fd-1

**Comment by gort818** [comment]
I have the same card, will check this. Can you confirm that  sysfs reports the correct GPU clock.

**Comment by DoomPenguin9** [comment]
> I have the same card, will check this. Can you confirm that sysfs reports the correct GPU clock.

Yes, it reports the correct values. `sudo cat /sys/kernel/debug/dri/0/amdgpu_pm_info` yields

````
GFX Clocks and Power:
	1249 MHz (MCLK)
	2827 MHz (SCLK)
	1845 MHz (PSTATE_SCLK)
	1249 MHz (PSTATE_MCLK)
	1012 mV (VDDGFX)
	334.0 W (average GPU)

GPU Temperature: 56 C
GPU Load: 100 %
MEM Load: 18 %

**Comment by gort818** [comment]
Ok I can confirm the issue, doesn't look like we are doing anything wrong, `current_gfxclk` in gpu_metrics seems to be always @ 1200 mhz.

`current_gfxclk` looks to be supported here https://elixir.bootlin.com/linux/latest/source/drivers/gpu/drm/amd/pm/swsmu/smu13/smu_v13_0_0_ppt.c#L1338

Will make a report to amdgpu devs.

**Comment by DoomPenguin9** [comment]
MangoApp from the AUR works fine and shows the correct GPU clock.

**Comment by Kastor16** [comment]
I have tried both mangohud and mangohud-git from AUR and they both still show 1200 mhz at all times for me.
Memory clock isn't displayed at all either. I'm running a 7900 XT on Arch.

**Comment by gort818** [comment]
memory clock should work, make sure to use the param `vram`  and `gpu_mem_clock`

**Comment by Kagukara** [comment]
Both [mangohud](https://aur.archlinux.org/packages/mangohud) and [mangohud-git](https://aur.archlinux.org/packages/mangohud-git) on the AUR have the `gpu_core_clock` stuck at 1200MHZ still.

OS: Arch Linux 
Kernel: 6.3.1-arch1-1 
GPU: AMD ATI Radeon RX 7900 XTX
Mesa Version: Mesa 23.2.0-devel (git-11b1a37553)
MangoHUD version: v0.6.9-1-40-g3daf7d7

**Comment by flightlessmango** [comment]
fixed here 6abc794b42edc1a397448ce4330e8fd164a5c529
